static/css/explore_landing.scss

```diff
@@ -67,6 +67,25 @@
     }
   }
 
+  .explore-title {
+    h1 {
+      margin: 0;
+      color: var(--gm-3-ref-neutral-neutral-20, #303030);
```

Comment:
Note for later: we should extract & save all the figma CSS variables and save them in our `base.scss` file

Reply:
Ack.